### ISSUE TITLE
 Overlaying scenes with postprocessing in r167 is broken

### ISSUE THREAD

**Issue by Spiri0** [mrdoob/three.js#29087]
### Description

I create two scenes and two cameras. The first camera is a perspective camera and the second is an orthographic camera for the second scene. The second scene is rendered with the orthographic camera and should be rendered over the first scene. In this way, UI elements can be easily used in an app. Until r166 it worked like in the codePen example.

footnote:
I try to create issues here as rarely as possible because I have an idea of ​​how much work all the issues are. I often spend days trying to figure out what could be wrong when it doesn't work.
I'm excited about the three.js project. I have completely switched to webgpu and only work with the webgpu world in three.js. The elegance with which it all works makes me realize how much energy and passion there is from all the developers who work on it.

### Reproduction steps

I think the code examples on jsfiddle from you Mugen and mine from codePen are small enough, so I refer to the links here

### Code

Try to use ```postProcessing.render();``` instead of  ```renderer.render(scene, camera);``` in the codePen example.
It shouldn't make any difference in the display on the screen. But then you only see the uiscene and no longer both scenes.

render function from the codePen example:
```
function render() {
  requestAnimationFrame( render );
   
  renderer.autoClearColor = true;
  
  renderer.render(scene, camera);
  //postProcessing.render();
  
  renderer.autoClearColor = false;
  
  renderer.render(uiscene, uicamera);  
}
```

### Live example

From @Mugen87
https://jsfiddle.net/21gfet07/1/

From me
https://codepen.io/Spiri0/pen/bGPrdzE

### Version

167

**Comment by Mugen87** [comment]
tl;dr; It seems clear operations after `postProcessing.render();` do not work correctly.

**Comment by Mugen87** [comment]
`postProcessing.render();` renders directly to screen and not into the intermediate `frameBufferTarget` which is maintained in `Renderer` for tone mapping in color space conversion. This has been change in `r167` via #28909.

So if you call `clearDepth()` after `postProcessing.render();` and the output color space is `SRGBColorSpace`, it clears the depth of an empty `frameBufferTarget` and copies the empty framebuffer to the screen.

For the same reason, a subsequent `renderer.render()` calls overwrite whatever post processing has produces. The bit that controls the clear is shown below:

https://github.com/mrdoob/three.js/blob/e6edc5c1ae2434afaf53250c9fef8727ef56104d/src/renderers/common/Renderer.js#L1004-L1020

You can verify the root cause  by forcing no output color space and tone mapping in the scene: https://jsfiddle.net/86zs1p42/

**Comment by Mugen87** [comment]
This issue is indeed tricky. 

Basically the renderer assumes `frameBufferTarget` always holds the complete data of the scene and tone mapping and color space conversion happens at the very end of the rendering pipeline. As discussed before, this premise is incorrect since depending on how you render the scene you want to tone map and convert color spaces at a different point in time. 

So the current combination of the OPs code is invalid. When `PostProcessing` is used (or any custom workflow that writes data directly to the default framebuffer), subsequent calls of `renderer.render()` do not work correctly because the internal `frameBufferTarget` holds empty data. `PostProcessing` already rendered to screen and outputted the colors in the output color space.

**Comment by Mugen87** [comment]
I'm not marking this as a regression since the state in `r166` was incorrect in the first place. The workflows are more correct in `r167` however certain rendering combinations are not supported so far.

**Comment by Spiri0** [comment]
Then the fact that it worked until r166 was a happy coincidence?

**Comment by Mugen87** [comment]
It was intended to support this use case but assumptions were made which were too simplified. Besides, we don't have a single example that uses the code setup from this issue. So I would say we have not thought about the use case when somebody wants to call `renderer.render()` after `PostProcessing.render()`.

**Comment by Mugen87** [comment]
Also notice that even your original codepen works if you exchange the output color space from `SRGBColorSpace` to `LinearSRGBColorSpace`. Of course the colors are not right but it demonstrates that the tone mapping and output color space step is the problematic bit.

**Comment by Spiri0** [comment]
It feels like I'm still pretty much alone in the community with my massive use of WebGPU and the node system. Thinking through all the options in advance is not possible or would take an unnecessary amount of time and energy. It works very well the way it is handled 👍

**Comment by sunag** [comment]
@Spiri0 Just as an additional solution, you can use `pass()` for your UI, it simplifies things a bit and you can blend with the already rendered scene however you want, like using `overlay()` instead of `mix()` as in the example below. You can remove all `renderer.clear()` from your setup.

```js
//renderer.setClearColor(0x000000); // ignore
//renderer.autoClear = false; // ignore

const scenePass = pass(scene, camera);
const uiPass = pass(uiscene, uicamera);

postProcessing.outputNode = mix( scenePass, uiPass, uiPass.a );
```

**Comment by Spiri0** [comment]
@Mugen87 I'll test it with the ```mix``` as you recommend this evening because I'm curious now about this alternative. If you and @sunag's ```overlay``` consider this to be the more efficient way then it is the new standard for me.

**Comment by sunag** [comment]
https://github.com/mrdoob/three.js/issues/29087#issuecomment-2275811214 This approach is more powerful, you can mix your GUI with different blending modes and create different effects.

**Comment by Spiri0** [comment]
I was curiose because instead of the scenePass I use a shader

```
//illustration
const shaderParams = {
  diffuse: myScenePassTextureNode,
  depth: myScenePassDepthTextureNode
}

mix( myShader(shaderParams), scenePassUI, scenePassUI.a );
```

With ```mix``` it looks right. For ```overlay``` I have to look in the node system to see how I have to use it because with ```overlay``` I get mixed scenes instead of overlaid like with ```mix```.
The bottom line is that it works well with a custom shader which can read in multiple pass textures instead of a single scenePass.
@Mugen87 what should happen with the issue? The new way does not require manual render clears. And I get along very well with that.

**Comment by Mugen87** [comment]
I would suggest we close the issue for now and see how other develoeprs approach the API. We can reopen the issue if the solution outlined in https://github.com/mrdoob/three.js/issues/29087#issuecomment-2275811214 is not acceptable for certain use cases.
